rfc/rfc0XX-use-case-list.md

```diff
@@ -90,20 +90,24 @@ If one or more presentations are not valid, it SHOULD NOT reject the other prese
 
 ### 3.4 List pruning
 
-To only keep relevant presentations on the list, maintainers MUST remove entries after a specified period of time.
-For instance, a presentations could be removed a week after they were added.
+To only keep valid presentations on the list, maintainers MUST remove presentations which `exp` (expiration) has passed from the list.
 
-Clients MUST re-submit their registration before that period of time has passed, if they want to keep their presentation on the list (once a day, for instance).
+Clients that wish to retain their presentation MUST re-submit their registration before that period of time has passed, if they want to keep their presentation on the list (once a day, for instance).
 
-Maintainers MUST remove presentations which ``expirationDate`` has passed from the list.
+Maintainers SHOULD remove presentations which contains credentials that have been revoked.
+
+### 3.5. Removing entries
+
+TODO: Clients may remove a credential subject by performing an HTTP DELETE to the list endpoint, where the body contains an empty Verifiable Presentation signed by the credential subject.
+TODO: If we support this: we need to serve a tombstone set as well (and also for revocations)
```

Comment:
If you add a lamport clock per entry and a tombstone list then you have the feature complete.